unity-renderer/Assets/UIComponents/Scripts/Components/Image/ImageComponentView.cs

```diff
@@ -1,5 +1,6 @@
 using DCL.Helpers;
 using System;
+using UnityEditor.Sprites;
```

Comment:
this wont work on build. Are we using it?

Reply:
nice catch. It was unused `using` :) Probably, I was using some Editor API to get rid of the Sprite MeshGeneration call but then found another way. Will remove it